pkg/archive/xattr_unsupported.go

```diff
@@ -0,0 +1,13 @@
+//go:build !linux && !darwin && !freebsd && !netbsd
+
+package archive
+
+// lgetxattr is not supported on Windows.
```

Comment:
nit: this file is now no longer windows-only; if you're still updating the PR, it's probably fine to just remove the comment (should be clear from the filename that these are stubs)